### ISSUE TITLE
No support for no table height

### ISSUE THREAD

**Issue by MikeWarren2014** [mckervinc/react-fluid-table#6]
I am using this project because I am not trying to hard-code in any table height. (I shouldn't have to hard-code **anything** but that's another story.)

I [see in the docs](https://mckervinc.github.io/react-fluid-table/#/props) that `tableHeight` is not required. However, [per this CodeSandbox](https://codesandbox.io/s/react-fluid-table-demo-tm3lo), it is (table won't render without it.

**Comment by mckervinc** [comment]
@MikeWarren2014 thanks for letting me know!

So the first thing I noticed was that there were some other unrelated errors in the CodeSandbox in `Table.js`. here's the updated version:

```jsx
import React from "react";
import { Table } from "react-fluid-table";

const data = [{ id: 1, firstName: "Mike", lastName: "Warren", email: "Test" }];

const columns = [
  { key: "id", header: "ID" },
  { key: "firstName", header: "First" },
  { key: "lastName", header: "Last" },
  { key: "email", header: "Email" }
];

export const Example = () => (
  <Table className="table" data={data} columns={columns} tableHeight={400} />
);

```

The import was missing curly braces, and I changed employee to be a basic JS object.

And the way `tableHeight` currently works is a little tricky. It's not required if the component that wraps the table has a set height. So for example, it wouldn't be required in this case:
```
<div style={{height: "400px"}}>
 <Table {...props} />
</div>
```

It's something that I am actively trying to solve, and I'll try and get a fix out for that in my next update

**Comment by mckervinc** [comment]
@MikeWarren2014 this should be fixed in version 0.3.2. The height should work as expected (you don't need to specify a height on the parent or on the component if you don't want to).

See the updated documentation
